docs/sources/setup-grafana/configure-docker.md

```diff
@@ -57,48 +91,217 @@ The following settings are hard-coded when launching the Grafana Docker containe
+
+Grafana ships with built-in support for the [Amazon CloudWatch datasource]({{< relref "../datasources/aws-cloudwatch/" >}}). To configure the data source, you must provide information such as the AWS ID-Key, secret access key, region, and so on. You can use Docker secrets as a way to provide this information.
+
+Example:
+
+The following example defines how to use Grafana environment variables via Docker Secrets for the AWS ID-Key, secret access key, region, and profile.
 
 ```bash
-docker run -d \
--p 3000:3000 \
---name=grafana \
--e "GF_AWS_PROFILES=default" \
--e "GF_AWS_default_ACCESS_KEY_ID=YOUR_ACCESS_KEY" \
--e "GF_AWS_default_SECRET_ACCESS_KEY=YOUR_SECRET_KEY" \
--e "GF_AWS_default_REGION=us-east-1" \
+docker run -d
+-p 3000:3000
+--name=grafana
+-e "GF_AWS_PROFILES=default"
+-e "GF_AWS_default_ACCESS_KEY_ID=YOUR_ACCESS_KEY"
+-e "GF_AWS_default_SECRET_ACCESS_KEY=YOUR_SECRET_KEY"
+-e "GF_AWS_default_REGION=us-east-1"
 grafana/grafana-enterprise
 ```
 
-You may also specify multiple profiles to `GF_AWS_PROFILES` (e.g.
-`GF_AWS_PROFILES=default another`).
+You can also specify multiple profiles to `GF_AWS_PROFILES` (for example, `GF_AWS_PROFILES=default another`).
 
 Supported variables:
 
 - `GF_AWS_${profile}_ACCESS_KEY_ID`: AWS access key ID (required).
 - `GF_AWS_${profile}_SECRET_ACCESS_KEY`: AWS secret access key (required).
 - `GF_AWS_${profile}_REGION`: AWS region (optional).
+
+## Troubleshoot a Docker deployment
+
+By default, the Grafana log level is set to `INFO`, but you can increase the log level to `DEBUG` mode when you try to reproduce a problem.
+
+For more information about logging, refer to [logs]({{< relref "./configure-grafana/#log" >}}).
+
+### Increase log level using the Docker run (CLI) command
+
+To increase the log level to `DEBUG` mode, add the environment variable `GF_LOG_LEVEL` to the command line.
+
+```bash
+docker run -d -p 3000:3000 --name=grafana
+-e "GF_LOG_LEVEL=DEBUG"
+grafana/grafana-enterprise
+```
+
+### Increase log level using the Docker Compose
+
+To increase the log level to `DEBUG` mode, add the environment variable `GF_LOG_LEVEL` to the `docker-compose.yaml` file.
+
+```yaml
+version: '3.8'
+services:
+  grafana:
+    image: grafana/grafana-enterprise
+    container_name: grafana
+    restart: unless-stopped
+    environment:
+      # increases the log level from info to debug
+      - GF_LOG_LEVEL=debug
+    ports:
+      - '3000:3000'
+    volumes:
+      - 'grafana_storage:/var/lib/grafana'
+volumes:
+  grafana_storage: {}
+```
+
+### Validate Docker Compose YAML file
+
+The chance syntax errors to appear in a YAML file increases as the file becomes more complex. You can use the following command to check for syntax errors.
+
+```bash
+docker compose config docker-compose.yaml
+```
+
+The output of the command highlights errors in specific lines. Else it will output the content of the `docker-compose.yaml` file in detailed YAML format.
```

Comment:
@usmangt , I'm don't understand the sentence that begins with "Else it...". can you please clarify what this means? Thanks!